scripts/apps/authoring/authoring/controllers/ChangeImageController.ts

```diff
@@ -73,6 +73,9 @@ export function ChangeImageController($scope, notify, _, api, $rootScope, $q) {
     $scope.crops = {
         isDirty: false,
     };
+    $scope.toggleShowMetadata = (value?) => {
+        $scope.showMetadata = value != null ? value : !$scope.showMetadata;
```

Comment:
Any reason why conditional handling exists here? The places that call this function without value have "Close details" /  "Open metadata" tooltips. Why not pass true/false and get rid of the conditional here?